cmake/LibFindMacros.cmake

```diff
@@ -3,167 +3,36 @@
 # Maintained at https://github.com/Tronic/cmake-modules
```

Comment:
Does cmake 3.x include most of this now, or is it more like we don't use all/most of it?

Reply:
I don't think we need it.

Reply:
Elaborated on the commit message/pr description